rules/windows/defense_evasion_process_termination_followed_by_deletion.toml

```
@@ -0,0 +1,45 @@
+[metadata]
+creation_date = "2020/11/04"
+maturity = "production"
+updated_date = "2020/11/04"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a process termination event quickly followed by the deletion of its executable file. Malware tools and other non-native
+files dropped or created on a system by an adversary may leave traces to indicate to what occurred. Removal of these files can occur during an intrusion, or as part of a post-intrusion process to minimize the
+adversary's footprint.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Process Termination followed by Deletion"
+risk_score = 43
```

```suggestion
risk_score = 47
```

Our other medium severity rules have a risk score of 47 if you want to align this one with those.